libzmix/bbs/src/prover.rs

```diff
@@ -123,8 +123,8 @@ impl Prover {
     /// * `nonce` - a SignatureNonce
     /// * `claims` - a vec of strings the prover wishes to include in the challenge (may be empty)
```

Comment:
This comment may be updated now? I think it's supposed to be an optional slice of bytes. Not recognizing if these types are the same. I'm a bit "rusty" at reading rust

![image](https://media.giphy.com/media/fWgPzLGeI4okiaG8QT/giphy.gif)

Reply:
good catch, fixed in [df1086c](https://github.com/hyperledger/ursa/pull/120/commits/df1086cfaad044d82c79eb6a1f02749c3a50f2e7)